README.md

```diff
@@ -23,24 +27,25 @@ The repository consists of the following Java classes:
 The repository consists of the following scripts and resources: 
 
 1. src/test/resources data files for the test classes 
-2. dataflow-staging-policy.json - a policy for expiring objects in the bucket used for staging the dataflow jobs
-3. run-simple.example - example script for running the simple pipeline using maven
-4. run-nested.example - example script for running the nested pipeline using maven
-5. pom.xml - maven build script 
+1. dataflow-staging-policy.json - a policy for expiring objects in the bucket used for staging the dataflow jobs
+1. run.sh - example script for running the pipelines using maven
+1. pom.xml - maven build script 
  
```

Comment:
To the extent this impacts the published doc lets tee up that doc soon.  Is it ready to roll?

Reply:
after a review, yes!